static/_redirects

```diff
@@ -375,6 +379,21 @@ https://0-20-0.docs.pomerium.com/category/guides https://0-20-0.docs.pomerium.co
 /docs/reference/branding/primary-color /docs/reference/branding
 /docs/reference/branding/secondary-color /docs/reference/branding
 
+# TLS
+/docs/reference/routes/tls-server-name /docs/reference/routes/tls
+/docs/reference/routes/tls-upstream-server-name /docs/reference/routes/tls
+
+# Metrics
```

Comment:
I think there are three more redirects that should be added here, based on https://github.com/pomerium/documentation/commit/ae81e48ef0cbaccdc9dd657d46b3322294efe963.